packages/ng-schematics/src/schematics/utils/types.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2022 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export enum TestingFramework {
+  Jasmine = 'jasmine',
+  Jest = 'jest',
+  Mocha = 'mocha',
```

Comment:
Let's also support the node's built-in test framework

Reply:
(in next PRs)

Reply:
I will add it in the types for visibility of thing we are looking into supporting.